File: Programming Basics with C#/01. First Steps In Coding - Lab/09. Yard Greening.cs
using System;

namespace YardGreening
{
    class Program
    {
        static void Main(string[] args)
        {
            double sqMeter = double.Parse(Console.ReadLine());
            double sqMeterPrice = sqMeter * 7.61;
            double discount = sqMeterPrice * 0.18;
            double finalPrice = sqMeterPrice - discount;

            Console.WriteLine($"The final price is: {finalPrice} lv.");
            Console.WriteLine($"The discount is: {discount} lv.");
        }
    }
}

File: Programming Basics with JavaScript/09. Yard Greening.js
function yardGreening(input){
    let sqrm = input[0];
    let sqrmSum = sqrm * 7.61;
    let discount = 0.18 * sqrmSum;
    let total = sqrmSum - discount;
    
    console.log(`The final price is: ${total} lv.`);
    console.log(`The discount is: ${discount} lv.`);
}
